packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css

```diff
@@ -258,10 +258,12 @@
 .dmn-decision-table-container .decision-table-name {
   position: relative;
   display: inline-block;

```

Comment:
Since the parent has `display: flex`, I don't think we need this, do we? (aren't we mixing concepts then?)

Reply:
Yeah, it makes sense. I've pushed a change for that plug a few `span -> div` changes as we use blocks.